sandboxnu/mfa-form-automator
# Jwt cookie

## Description/Problem

Support JWT cookies for auth.

## Solution

Auth

## Dependencies

[X] This PR adds new dependencies

## Testing

Please describe how you tested this PR (manually and/or automatically)
Provide instructions so we can reproduce, and include screenshots of UI changes.

- Manual Tests?
  - Check login/logout and see if cookie headers are being sent
- Unit/Integration/E2E Tests?
  - CI passing
